app/models/job/Job.scala

```diff
@@ -341,7 +341,7 @@ class JobService @Inject()(wkConf: WkConf,
       Json.obj(
         "id" -> job._id.id,
         "command" -> job.command,
-        "commandArgs" -> (job.commandArgs - "webknossos_token" - "user_auth_token"),
+        "commandArgs" -> (job.commandArgs - "webknossos_token"),
```

Comment:
```suggestion
        "commandArgs" -> (job.commandArgs - "webknossos_token" - "user_auth_token"),
```
This should remain here to ensure we are not leaking tokens used in existing jobs (otherwise a DB migration would be required removing them, that would be fine by me too)